src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2383,8 +2383,13 @@ public function get_test_authorization_header() {
 	 * @return array The test result.
 	 */
 	public function get_test_page_cache() {
+		$caching_url  = apply_filters( 'site_status_persistent_page_cache_url', 'homepage' );
```

Comment:
```suggestion
		/** This filter is documented in wp-admin/includes/class-wp-site-health.php */
		$caching_url  = apply_filters( 'site_status_persistent_page_cache_url', home_url( '/' ) );
```

Filter use URL so we can't add string here.